### PR TITLE
show chain id

### PR DESCRIPTION
**Pull request by Vadim121197**

### CODE REVIEW

File: apps/extension/src/clients/index.ts [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-import { createChannelTransport, ChannelClientLabel } from '@penumbra-zone/transport';
-import { ClientConnectionManager } from '@penumbra-zone/transport/src/chrome-runtime/client-connection-manager';
-
-export const grpcClient = createPromiseClient(
-  ViewProtocolService,
-  createChannelTransport(
-    ViewProtocolService,
-    ClientConnectionManager.init(ChannelClientLabel.Extension),
-  ),
-);
```
Comment (grod220) [inline apps/extension/src/clients/index.ts:12]: It looks like you found a workaround, but this should be able to work. There is something deeper wrong. Can you do more debugging to see why the popup is not able to use this grpc client?

Comment (Vadim121197) [reply apps/extension/src/clients/index.ts:12]: Ok

Comment (turbocrime) [reply apps/extension/src/clients/index.ts:12]: sorry, this looks like my fault. missed this internal client init in one of the refactors during review. intra-extension transport init requires one more step. i'll correct this in a moment

**Comment by turbocrime** [comment]
Screenshot 2023-12-12 at 15 34 57

**Review by turbocrime — APPROVED** [review]
corrected init, cleaned up problematic unused checks in the background manager. this is working now

**Pull request merged**